.circleci/config.yml

```diff
@@ -404,6 +416,106 @@ references:
     - Demisto SDK Nightly - Run against Cortex XSOAR:
         context: sdk_nightly_env
 
+  bucket_upload_jobs: &bucket_upload_jobs
+    - Setup Environment
+    - Create Instances:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Run Unit Testing And Lint:
```

Comment:
Need to make sure unittests and validation are running on all packs to verify that the content is indeed valid

Reply:
it is running on all content packs